feature_layers/feature-collection-layer-from-portal/src/main/java/com/esri/samples/feature_collection_layer_from_portal/FeatureCollectionLayerFromPortalSample.java

```diff
@@ -0,0 +1,151 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.esri.samples.feature_collection_layer_from_portal;
+
+import com.esri.arcgisruntime.data.FeatureCollection;
+import com.esri.arcgisruntime.layers.FeatureCollectionLayer;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.portal.Portal;
+import com.esri.arcgisruntime.portal.PortalItem;
+
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.Alert;
+import javafx.scene.control.Button;
+import javafx.scene.control.TextField;
+import javafx.application.Application;
+import javafx.scene.layout.StackPane;
+import javafx.stage.Stage;
+
+
+public class FeatureCollectionLayerFromPortalSample extends Application {
+
+    private MapView mapView;
+    private ArcGISMap map;
+    private Portal portal;
+    private FeatureCollectionLayer featureCollectionLayer;
+    private FeatureCollection featureCollection;
+    private TextField input;
+
+    @Override
+    public void start(Stage stage) {
+        // set up the scene
+        try {
+            // create stack pane and application scene
+            StackPane stackPane = new StackPane();
+            Scene scene = new Scene(stackPane);
+
+            // set title, size, and add scene to stage
+            stage.setTitle("Feature Collection Layer From Portal");
+            stage.setWidth(800);
+            stage.setHeight(700);
+            stage.setScene(scene);
+            stage.show();
+
+            // create amp and set it to be displayed in this view
+            map = new ArcGISMap(Basemap.createOceans());
+            // create a view for this ArcGISMap and set ArcGISMap to it
+            mapView = new MapView();
+            mapView.setMap(map);
+
+            // create text field to input user's own coordinate string
+            input = new TextField();
+            input.setMaxWidth(250);
+            input.setText("32798dfad17942858d5eef82ee802f0b");
+
+            // create button to perform action
+            Button fetchFromPortal = new Button("Open from portal item");
+            fetchFromPortal.setMaxWidth(250);
+
+            /**
+             * Action event on button and validation of empty Text Filed
+             */
+            fetchFromPortal.setOnAction(e -> {
+
+                if (!"".equals(input.getText())) {
+                    fetchFeatureCollectionFromPortal();
+                } else {
+                    new Alert(Alert.AlertType.ERROR, "Portal Itemid is empty. Please enter a portal item id.").show();
```

Comment:
We could make this prompt a bit more uniform in terms of formatting
```suggestion
                    new Alert(Alert.AlertType.ERROR, "Portal Item ID is empty. Please enter a Portal Item ID.").show();
```